docs/go/getting-started.md

```diff
@@ -81,25 +81,25 @@ This file declares the `greet.v1` Protobuf package, a service called
 response structures. These package, service, and method names will reappear
 soon in our HTTP API's URLs.
 
-## Generate code
+## Generate code using Buf v2
 
 We're going to generate our code using [Buf][buf], a modern replacement for
 Google's protobuf compiler. We installed Buf earlier, but we also need a few
 configuration files to get going. (If you'd prefer, you can skip this section
 and use `protoc` instead &mdash; `protoc-gen-connect-go` behaves like any other
 plugin.)
 
-First, scaffold a basic [`buf.yaml`][buf.yaml] by running `buf mod init`. Next,
+First, scaffold a basic [`buf.yaml`][buf.yaml] by running `buf config init`. Next,
```

Comment:
Command "init" is deprecated, use "buf config init" instead. However, "buf mod init" will continue to work.